tools/please_go/generate/generate.go

```diff
@@ -149,6 +149,9 @@ func (g *Generate) generateAll(dir string) error {
 			if info.Name() == "testdata" {
 				return filepath.SkipDir
 			}
+			if path != dir && strings.HasPrefix(info.Name(), "_") {
+				return filepath.SkipDir
+			}
```

Comment:
THis was just another small fix I had laying around to stop us descending into `_blah` dirs. These are almost always some internal nonsense that we don't need to generate BUILD files for.